conf/postfix/main.cf

```diff
@@ -99,7 +99,7 @@ inet_interfaces = all
 message_size_limit = 35914708
 
 # Virtual Domains Control
-virtual_mailbox_domains = ldap:/etc/postfix/ldap-domains.cf
```

Comment:
So shall we get rid of that file from the regenconf ? https://github.com/YunoHost/yunohost/blob/dev/conf/postfix/plain/ldap-domains.cf

Reply:
Done by emptying the file